packages/uniforms-bridge-zod/__tests__/ZodBridge.ts

```diff
@@ -89,6 +89,29 @@ describe('ZodBridge', () => {
       expect(error?.issues?.[0]?.message).toBe(errorMessage);
     });
 
+    it('works with chained refined schema', () => {
+      const firstErrorMessage = 'Different values';
+      const secondErrorMessage = 'Different moduloes of 2 and 3';
```

Comment:
I believe `moduloes` it's not proper word. Just write `Different results` instead of `Different moduloes of 2 and 3`